libsel4vm/src/arch/x86/vmcs.c

```diff
@@ -62,8 +97,13 @@ void vm_vmcs_init_guest(vm_vcpu_t *vcpu)
     vm_vmcs_write(vcpu->vcpu.cptr, VMX_GUEST_RFLAGS, BIT(1));
     vm_vmcs_write(vcpu->vcpu.cptr, VMX_GUEST_SYSENTER_ESP, 0);
     vm_vmcs_write(vcpu->vcpu.cptr, VMX_GUEST_SYSENTER_EIP, 0);
-    vcpu->vcpu_arch.guest_state->machine.control_ppc = VMX_CONTROL_PPC_HLT_EXITING | VMX_CONTROL_PPC_CR3_LOAD_EXITING |
-                                                       VMX_CONTROL_PPC_CR3_STORE_EXITING;
+
+#ifdef CONFIG_ARCH_X86_64
+    vm_vmcs_init_64_bit_guest(vcpu);
+#else
+    vm_vmcs_init_32_bit_guest(vcpu);
```

Comment:
There are example apps that run a 32bit guest on a 64 bit host.  Is this no longer something that we will support?

Reply:
Yep, just found out as well when fixing some builds. I need to make extra changes to the various repositories to allow 32-bit guest on 64-bit host.

Reply:
Have these changes been made?